procedures/katello/branch.md.erb

```diff
@@ -59,6 +59,8 @@
 - [ ] Push version bumps to <%= develop %>-master using <%= rel_eng_script('develop_branch_push') %>
 - [] Bump Katello related packages to their next versions
   - [] [packaging](https://github.com/theforeman/foreman-packaging/pull/8729/files)
+- [ ] Set GitHub Actions to use the new branch
+  - [ ] [React tests](https://github.com/Katello/katello/pull/11053)
```

Comment:
I'd prefer a specific command. For example:
```suggestion
- [ ] Set GitHub Actions to use the new branch: `sed -i '/foreman_version/ s/develop/<%= foreman_version %>-stable/' .github/workflows/*.yml`
```

Note this relies on adding `foreman_version` as suggested in https://github.com/Katello/katello/pull/11053/files#r1657348705.